_studio/shared/umc/codec/color_space_converter/include/umc_video_resizing.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include "umc_defs.h"
+
+#if defined (UMC_ENABLE_VPP_RESIZE)
```

Comment:
I do not see anywhere that you have defined UMC_ENABLE_VPP_RESIZE. If so, the code under it is _dead_ and should be dropped...

Questions:
1. Have you actually checked how SW jpeg decoder is running in open source? Did you run any tests? What is passing and what is not?
2. Could we try to break SW jpeg enabling support into 2 steps based on this UMC_ENABLE_VPP_RESIZE?

I very much don't like having UMC_ENABLE_VPP_RESIZE at all. From my perspective our SW_FALLBACK should be enough.

Reply:
`UMC_ENABLE_VPP_RESIZE` is not defined in this patch. But resize topic is interesting.

Manually I run
jpegd_conf jpegd_difresol jpegd_multi jpegd_multiscan jpegd_reli jpegd_resize jpegd_threadsafety mjpegd_conf mjpegd_double_rotate mjpegd_multi mjpegd_reli mjpegd_reposition mjpegd_resize mjpegd_threadsafety v30b5_jpegd_rotate v30b5_mjpegd_rotate jpege_func jpege_multi jpege_reli jpege_resize jpege_robust jpege_threadsafety mjpege_func mjpege_reli mjpege_resize mjpege_robust mjpege_threadsafety

Except for some flaky failures, new failures are jpegd_resize 0155 0159. That functionality is depends on `MFX_ENABLE_HW_ONLY_VPP` which is presented in closed-source only. This was discussed verbally and current status is "let's postpone it".

Reply:
Hm. Why we need the code which is under UMC_ENABLE_VPP_RESIZE then? If we don't exercise it within Jpeg, then it should go away.

Reply:
So, removed.